### ISSUE TITLE
Wrong `ShutterSpeedValue` on some images

### ISSUE THREAD

**Issue by EricRovell** [mattiasw/ExifReader#201]
### Description

Hi, thanks for a great package!

I have some issues with `ShutterSpeedValue` value. Lightroom and exiftool shows the correct value, but when I try to get the data using this package, I get some really strange values.

For example, when the correct value is `6.0` I get this data instead:

```js
    "ShutterSpeedValue": {
      "id": 37377,
      "value": [
        -2584963,
        1000000
      ],
      "description": "1/0"
    },
```

Here is the [example ](https://files.catbox.moe/3408yg.JPG) of the image where the shutter speed value is wrong.

- ExifReader version: 4.5.0
- Node version: Node 16.10.0

**Comment by mattiasw** [comment]
Hi! I think I have a fix. I just have to test it some more. I will continue tomorrow. It had to do with when the shutter speed was longer than one second. Then the value was presented in the wrong way (i.e. in `description`; `value` is correct but it's not very helpful unless one use the correct formula to get an understandable representation).

**Comment by mattiasw** [comment]
Published as version 4.5.1.

**Comment by EricRovell** [comment]
@mattiasw Thank you for a fix!
